exchange/src/randomness/brownian.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file brownian.hpp
+ * @author Andrew Li (andrewli@u.northwestern.edu)
+ * @brief Brownian motion randomness to simulate market chaos
+ * @version 0.1
+ * @date 2024-01-12
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#pragma once
+
+#include <random>
+
+namespace nutc {
+
+namespace stochastic {
+
+class BrownianMotion {
+    double cur_value_;
+    std::mt19937 random_number_generator_;
+
+public:
+    // Default constructor for BrownianMotion, takes nothing
+    explicit BrownianMotion() : cur_value_(0)
```

Comment:
I don't think explicit is necessary here as there's no risk of implicit type conversions, but this is enough of a nitpick that I'll ignore it